src/ui/guests/guests.css

```
@@ -4,6 +4,9 @@
   --guest-rule: #8c3229;
   --guest-muted: #5b5449;
 
+  min-width: 0;
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)